### PR TITLE
Master with storer fixes

### PR DESCRIPTION
**Pull request by iulianpascalau**
same commits as #3562

### CODE REVIEW

File: storage/pruning/fullHistoryPruningStorer_test.go [hunk 1]
```diff
@@ -270,6 +272,7 @@ func TestFullHistoryPruningStorer_ConcurrentOperations(t *testing.T) {
 
 	startTime := time.Now()
 
+	_ = logger.SetLogLevel("*:DEBUG")
```
Comment (sasurobert) [inline storage/pruning/fullHistoryPruningStorer_test.go:275]: delete log

**Review by raduchis — APPROVED** [review]

**Review by SebastianMarian — APPROVED** [review]

**Review by AdoAdoAdo — APPROVED** [review]

File: storage/pruning/fullHistoryPruningStorer_test.go (hunk 1 above)
Comment (iulianpascalau) [reply storage/pruning/fullHistoryPruningStorer_test.go:275]: we have this on development. Let's remove-it from there

Comment (iulianpascalau) [reply storage/pruning/fullHistoryPruningStorer_test.go:275]: anyway, the test is deactivated

**Review by gabi-vuls — APPROVED** [review]
System test passed.
image

**Pull request merged**